src/main/java/school/lemon/changerequest/java/introduction/hw1/Task10.java

```diff
@@ -4,6 +4,6 @@
 public class Task10 {
 
     public static boolean containDigitTwo(int n) {
-        return false;
+        return Integer.toString(n).contains("2");
```

Comment:
For sure it is the most small, but not the most efficient solution. Operations with strings is very expensive. 
Please find out the better solution using operators from the lecture (`/`, `%`, `*`, etc).